docs/docs-content/clusters/data-center/vmware/permissions.md

```diff
@@ -7,186 +7,231 @@ sidebar_position: 60
 tags: ["data center", "vmware", "permissions"]
 ---
 
-The VMware vSphere user account that deploys host clusters require access to the following vSphere objects and
-permissions listed in the following table. Review the vSphere objects and privileges required to ensure each role is
-assigned the required privileges.
+The VMware vSphere user account that deploys host clusters and/or private cloud gateways requires all vSphere privileges
+listed in the following table on specific objects. A _Spectro root role_ must be created that contains each privilege in
+the table.
 
 ### Spectro Root Role Privileges
 
-The spectro root role privileges are only applied to root objects and data center objects. Select the tab for the
-vSphere version you are using to view the required privileges for the spectro root role.
+Select the tab for the vSphere version you are using to view the required privileges for the _Spectro root role_.
```

Comment:
```suggestion
A _Spectro root role_ must be created that contains each privilege in the following tables. Select the tab for the vSphere version you are using to view the required privileges.
```

Reply:
Resolved in eff3548d1103ad25895211ffc3fc1076aabc3de1